src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItem.scala

```diff
@@ -114,11 +114,13 @@ object EasyStageFileItem {
 
 
   def createFileSdo(sdoDir: File, parent: (String,String))(implicit s: FileItemSettings): Try[Unit] = {
+    assert(s.datastreamLocation.isDefined != s.file.isDefined,
```

Comment:
@rvanheest : Changed that back also. It turns out that `FileItemSettings` is also used to create folder items, which do not have this constraint.